### PR TITLE
Convert all ports backend to Laravel style ajax table

### PR DESCRIPTION
**Pull request by murrant**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
Ready.

File: includes/html/pages/ports/list.inc.php [hunk 1]
```diff
@@ -98,6 +98,9 @@ function formatUnits(units,decimals,display,base) {
     formatters: {
       'device': function (column, row) {
           return "<span class='alert-status " + row.status + "' style='float:left;margin-right:10px;'></span>" + row.device + "";
+      },
+      'port': function (column, row) {
+          return row.port
```
Comment (Jellyfrog) [inline includes/html/pages/ports/list.inc.php:103]: This doesn't seem necessary ?

**Comment by Jellyfrog** [comment]
This looks weird
Screenshot from 2021-09-01 18-20-38

File: includes/html/pages/ports/list.inc.php [hunk 2]
```diff
@@ -33,9 +33,9 @@
         <table id="ports" class="table table-condensed table-hover table-striped">
             <thead>
             <tr>
-                <th data-column-id="device" data-formatter="device">Device</th>
-                <th data-column-id="port"<?php echo $sort ?>>Port</th>
-                <th data-column-id="ifLastChange" data-converter="duration">Status Changed</th>
+                <th data-column-id="hostname" data-formatter="device">Device</th>
+                <th data-column-id="ifDescr"<?php echo $sort ?> data-formatter="device">Port</th>
```
Comment (Jellyfrog) [inline includes/html/pages/ports/list.inc.php:37]:
```suggestion
                <th data-column-id="ifDescr"<?php echo $sort ?> data-formatter="port">Port</th>
```

?

Comment (murrant) [reply includes/html/pages/ports/list.inc.php:37]: yeah, changed it to a formatter in one of the last commits

**Pull request merged**